index.html

```diff
@@ -3,7 +3,7 @@
   "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
 
 
-<meta http-equiv="refresh" content="0; url=http://statsmodels.github.io/dev/>
+<meta http-equiv="refresh" content="0; url=http://statsmodels.github.io/dev"/>
```

Comment:
It's strange because it was working for me locally. Anyway, my last commit got it. We should be good now!